widgets/contact/contact.php

```diff
@@ -785,6 +790,14 @@ function render_form_fields( $fields, $errors = array(), $instance ){
 		$field_ids = array();
 		$label_position = $instance['design']['labels']['position'];
 
+		$indicate_required_fields = $instance['settings']['required_field_indicator'];
+
+		if( ! empty( $indicate_required_fields ) ) {
+		    ?>
+            <p><em><?php esc_html_e( 'Fields marked with * are required', 'so-widgets-bundle' ) ?></em></p>
```

Comment:
I'd prefer to have this user configurable. We can make this text the default, but this is something that a lot of users are going to want to change (for translation and personal preference).